doc/examples/ubuntu.yaml

```diff
@@ -73,6 +73,10 @@ files:
 - path: /etc/machine-id
   generator: dump
 
+- path: /etc/profile
+  generator: copy
+  destination: /etc/user/profile
```

Comment:
I'd do it the other way around, `path` should always be the path inside the image, so you'd probably want a key called `source` instead.